Source/CarthageKit/Xcode.swift

```diff
@@ -222,20 +222,20 @@ public func schemesInProjects(projects: [(ProjectLocator, [String])]) -> SignalP
 
 /// Represents a platform to build for.
 public enum Platform: String {
-	/// Mac OS X.
-	case Mac
+	/// macOS.
+	case macOS = "Mac"
```

Comment:
The raw value should not be changed for backward compatibility since that is used in `Carthage/Build/Mac` dir.